tasktracker/config/models.py

```diff
@@ -6,20 +6,22 @@
 
 from tasktracker.models.model import Model
 
+BASE_DIR = Path().cwd().parent
+
 # Set the cache directory for Hugging Face transformers
-cache_dir = str(os.path.join(Path(__file__).resolve().parent.parent.parent, "disk1/"))
+cache_dir = str(os.path.join(BASE_DIR, "disk1/"))
 os.environ["TRANSFORMERS_CACHE"] = cache_dir
 os.environ["HF_HOME"] = cache_dir
 
 # Directory where model activation data will be stored
 activation_parent_dir = str(
-    os.path.join(Path(__file__).resolve().parent.parent.parent, "disk3/activations/")
+    os.path.join(BASE_DIR, "disk3/activations/")
 )
 
 # Directory where the dataset text files are stored
 text_dataset_parent_dir = str(
     os.path.join(
-        Path(__file__).resolve().parent.parent, "dataset_creation/dataset_sampled"
+        BASE_DIR, "dataset_creation/dataset_sampled"
```

Comment:
```suggestion
BASE_DIR, "tasktracker/dataset_creation/dataset_sampled"
```
To be consistent with the original implementation